BuildScript/BuildTargets/Testing.targets

```diff
@@ -178,21 +160,21 @@ docker run ^
 --volume $(_testAssemblyDirectory):$(_testAssemblyDirectory) ^
 --volume $(LogDirectory):$(LogDirectory) ^
 --entrypoint $(NunitFolderPathWithoutDotNet35Dependency)$(_nunitRunnerFileName) ^
-$(DotNetFrameworkRuntimeImageName) $(_nunitCallArgumentString)
+$(_executionRuntime) $(_nunitCallArgumentString)
```

Comment:
that looks iffy. we're suing _executionruntime once with docker-pull and once the testfilename. maybe we need two values here? want to use the "key" for the testresult filename, batching, etc and "image name" for the docker-pull.